contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -26,9 +26,9 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
      *************/
 
     // L2 tx gas-related
-    uint256 constant public MIN_ROLLUP_TX_GAS = 20000;
+    uint256 constant public MIN_ROLLUP_TX_GAS = 100000;
     uint256 constant public MAX_ROLLUP_TX_SIZE = 10000;
```

Comment:
Does making these constants save a lot of gas compared to making them constructor args? Its just going to be difficult to maintain having to make a commit if we want to change these params between different deployments. Then we will need to call them to remember what they are for the different deployments, unless we maintain a bunch of different git branches for each deployment which seems like a lot of extra work